screenshots/README.md

```diff
@@ -0,0 +1,171 @@
+
+### [bespin.png](bespin.png)
+
+![bespin.png](bespin.png "bespin.png")
+
+
+### [black.png](black.png)
+
+![black.png](black.png "black.png")
+
+
+### [darcula.png](darcula.png)
+
+![darcula.png](darcula.png "darcula.png")
+
+
+### [dark-colors.png](dark-colors.png)
+
+![dark-colors.png](dark-colors.png "dark-colors.png")
+
+
+### [dark-fruit-salad.png](dark-fruit-salad.png)
+
+![dark-fruit-salad.png](dark-fruit-salad.png "dark-fruit-salad.png")
+
+
+### [dark.png](dark.png)
+
+![dark.png](dark.png "dark.png")
+
+
+### [delt-dark.png](delt-dark.png)
+
+![delt-dark.png](delt-dark.png "delt-dark.png")
+
+
+### [epsilon.png](epsilon.png)
+
+![epsilon.png](epsilon.png "epsilon.png")
+
+
+### [fluffy.png](fluffy.png)
+
+![fluffy.png](fluffy.png "fluffy.png")
+
+
+### [gedit.png](gedit.png)
+
+![gedit.png](gedit.png "gedit.png")
+
+
+### [github.png](github.png)
+
+![github.png](github.png "github.png")
+
+
+### [himbeere.png](himbeere.png)
+
+![himbeere.png](himbeere.png "himbeere.png")
+
+
+### [inkpot.png](inkpot.png)
+
+![inkpot.png](inkpot.png "inkpot.png")
+
+
+### [mc.png](mc.png)
+
+![mc.png](mc.png "mc.png")
+
+
+### [metallic-bottle.png](metallic-bottle.png)
+
+![metallic-bottle.png](metallic-bottle.png "metallic-bottle.png")
+
+
+### [monokai.png](monokai.png)
+
+![monokai.png](monokai.png "monokai.png")
+
+
+### [notepad-plus-plus.png](notepad-plus-plus.png)
+
+![notepad-plus-plus.png](notepad-plus-plus.png "notepad-plus-plus.png")
+
+
+### [oblivion2.png](oblivion2.png)
+
+![oblivion2.png](oblivion2.png "oblivion2.png")
+
+
+### [pygments.png](pygments.png)
+
+![pygments.png](pygments.png "pygments.png")
+
+
+### [railcasts2.png](railcasts2.png)
+
+![railcasts2.png](railcasts2.png "railcasts2.png")
+
+
+### [retro.png](retro.png)
+
+![retro.png](retro.png "retro.png")
+
+
+### [screenshot-missing.png](screenshot-missing.png)
```

Comment:
This one shouldn't be here.

Reply:
You meant screenshot-missing, not `retro.png`, right?

Reply:
Yes